metta/common/formula/ModusPonensFormula.metta

```diff
@@ -0,0 +1,17 @@
+;; Formula for the modus ponens rule
+
+;; Import modules
+!(import! &self ../truthvalue/TruthValue.metta)
+!(import! &self ../Num.metta)
+
+(= (modus-ponens-formula $ptv $pqtv)
+    (let*
+        (($Pc (confidence $ptv))
+         ($PQc (confidence $pqtv))
+         ($Ps (mode $ptv))
+         ($PQs (mode $pqtv))
```

Comment:
@leungmanhin, I recommend extracting the strength and confidence via pattern matching inside the function definition which is considerably faster (and somewhat even more elegant) than using `confidence` and `mode`.

See below for example:

https://github.com/trueagi-io/hyperon-pln/blob/6a817b117cb21a748fcf5d0840d5a3309bb3bc3d/metta/common/formula/DeductionFormula.metta#L60-L75